api/clients/v2/verification/commitment_utils_test.go

```diff
@@ -35,58 +29,64 @@ func getRandomPaddedBytes(testRandom *random.TestRandom, count int) []byte {
 
 func TestComputeAndCompareKzgCommitmentSuccess(t *testing.T) {
 	testRandom := random.NewTestRandom(t)
-	randomBytes := getRandomPaddedBytes(testRandom, 1000)
+	randomBytes := getRandomPaddedBytes(testRandom, 100+testRandom.Intn(1000))
+
+	srsNumberToLoad := computeSrsNumber(len(randomBytes))
 
-	kzgVerifier, err := verifier.NewVerifier(getKzgConfig(), nil)
-	require.NotNil(t, kzgVerifier)
+	s1, err := kzg.ReadG1Points(g1Path, srsNumberToLoad, uint64(runtime.GOMAXPROCS(0)))
```

Comment:
guessing s1 is for srsG1? took me a while to understand what it stood for, consider changing to the more explicit form

Reply:
[c9bdbbe0](https://github.com/Layr-Labs/eigenda/pull/1063/commits/c9bdbbe04da6207372b61445cdc77efb379e4bc2)